Documentation/doc/biblio/geom.bib

```diff
@@ -152085,3 +152086,33 @@ @article{ledoux2014triangulation
   year={2014},
   publisher={Elsevier}
 }
+
+@article{schonhardt1928zerlegung,
+  title={{\"U}ber die zerlegung von dreieckspolyedern in tetraeder},
```

Comment:
zerlegung -> Zerlegung
dreieckspolyedern -> Dreieckspolyedern 
tetraeder -> Tetraeder

(The title is in German so a few more capitals...)

Reply:
I saw this warning as well.
I think this a side effect of the merging strategy used by CGAL for the overnight builds, so I didn't do a followup on it.

Reply:
Indeed. The later build [here](https://cgal.github.io/8186/test/Constrained_triangulation_3/index.html#CT_3_CDT) shows that the reference [5] is fin